tests/testAutopas/tests/selectors/tuningStrategy/PredictiveTuningTest.h

```diff
@@ -14,19 +15,69 @@
 
 class PredictiveTuningTest : public AutoPasTestBase {
  protected:
-  const autopas::Configuration configurationC01 = autopas::Configuration(
-      autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c01, autopas::LoadEstimatorOption::none,
-      autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled);
-  const autopas::Configuration configurationC08 = autopas::Configuration(
-      autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c08, autopas::LoadEstimatorOption::none,
+  /**
+   * Tunes for a few iterations (length of times vector) and checks whether all possible configurations were tuned.
+   * @param predictiveTuning The PredictiveTuning strategy.
+   * @param evidences Array of times to add to the evidences.
+   * @param returnConfigIndices The configurations with these indices are returned.
+   * @param iteration The current number of iterations, will be increased accordingly.
+   * @return Array of the configurations with indices returnConfigIndices.
```

Comment:
```suggestion
   * Simulates tuning of a few iterations (given by the length of evidence vector) and checks whether all possible configurations were tuned.
   * @param predictiveTuning The PredictiveTuning strategy object.
   * @param evidence Array of times to add to the evidence.
   * @param returnConfigIndices The configurations with these indices are returned.
   * @param iteration Number of the first iteration that will be simulated.
   * @return Array of the configurations with indices returnConfigIndices.
```

Rename variable `evidences` to `evidence` ("evidence" is always singular)

returnConfigIndices is way to complicated. Just return everything.